### ISSUE TITLE
Recolor doesn't work when background color change

### ISSUE THREAD

**Issue by xzchaoo** [andkulikov/Transitions-Everywhere#28]
I have my view's background changed from #SOME_ARGB to #ANOTHER_ARGB.
But transition Recolor doesn't work for it.

**Comment by andkulikov** [comment]
Hello. Can you please provide code so i can test it?

**Comment by xzchaoo** [comment]
Hello, I found the reason.
In the source code, in the method: Recolor.createAnimator
```java
        if (startBackground instanceof ColorDrawable && endBackground instanceof ColorDrawable) {
            ColorDrawable startColor = (ColorDrawable) startBackground;
            ColorDrawable endColor = (ColorDrawable) endBackground;
            if (startColor.getColor() != endColor.getColor()) {
                endColor.setColor(startColor.getColor());
                changed = true;
                return ObjectAnimator.ofObject(endBackground, "color",
                        new ArgbEvaluator(), startColor.getColor(), endColor.getColor());
            }
        }
```
here is a mistake : "endColor.setColor(startColor.getColor());".
It causes animating color from start color to same start color.

**Comment by andkulikov** [comment]
thanks! i will fix it in next release

**Comment by andkulikov** [comment]
released in 1.6.1
